packages/aws-cdk-lib/aws-stepfunctions/lib/activity.ts

```diff
@@ -26,6 +48,7 @@ export class Activity extends Resource implements IActivity {
    */
   public static fromActivityArn(scope: Construct, id: string, activityArn: string): IActivity {
     class Imported extends Resource implements IActivity {
+      kmsKey: kms.IKey | undefined;
```

Comment:
Not sure why this was added here? I don't see it being used in this class

Reply:
Will remove, I forgot what the issue was exactly but it was something to do with this not implementing the new field kmsKey but after making it optional like so and removing that line ^ it worked:

```
readonly kmsKey?: kms.IKey;
```